package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "agenda",
-  "version": "4.0.1",
+  "version": "4.1.0",
```

Comment:
Please revert back to 4.0.1. Bumping the version before an actual release is considered a bad practice.